components/cdc/src/delegate.rs

```diff
@@ -549,19 +708,17 @@ impl Delegate {
             } = cmd;
             if response.get_header().has_error() {
                 let err_header = response.mut_header().take_error();
-                self.mark_failed();
```

Comment:
Why we can remove the `mark_failed`s ?

Reply:
`mark_failed` will be called in the function which calls `on_batch`.